DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/JobProfile.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>JobProfile</id>
-    <version>1.0.27.2</version>
+    <version>1.0.28.1</version>
```

Comment:
We are in sprint 29, so this should be 1.0.Sprint.Revision = 1.0.29.1

Reply:
Changed to 1.0.29.1